webviz_subsurface/_utils/vector_calculator.py

```diff
@@ -0,0 +1,325 @@
+import sys
+from typing import List, Dict, Tuple, Union, Optional
+
+from pathlib import Path
+from uuid import uuid4
+import yaml
+import numpy as np
+import pandas as pd
+
+from webviz_config.common_cache import CACHE
+from webviz_subsurface_components import (
+    VectorCalculator,
+    ExpressionInfo,
+    ExternalParseData,
+    VariableVectorMapInfo,
+)
+
+if sys.version_info >= (3, 8):
+    from typing import TypedDict
+else:
+    from typing_extensions import TypedDict
+
+# JSON Schema for predefined expressions configuration
+# Used as schema input for json_schema.validate()
+PREDEFINED_EXPRESSIONS_JSON_SCHEMA: dict = {
```

Comment:
```suggestion
PREDEFINED_EXPRESSIONS_JSON_SCHEMA = {
```
Is type hint necessary here?